public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -0,0 +1,255 @@
+}
+
+
+export const SurveyAnswerRowComponent = (class extends React.Component<SurveyRowProps, SurveyRowState> {
+    constructor(props: SurveyRowProps, state: SurveyRowState) {
+        super(props);
+
+        this.state = {
+            drawerOpen: false,
+            subQuestion: {},
+            subAnswers: {},
+            subTitle: ''
+        };
+    }
+    surveyConceptId = this.props.stratum1;
+    hasSubQuestions = this.props.stratum7;
+
+
+    openDrawer(e) {
+        this.setState({
+            drawerOpen: !this.state.drawerOpen
+        });
+        if (this.hasSubQuestions === '1') {
+            this.getSubQuestions(1);
+        }
+    }
+    getSubQuestions(level: number) {
+        return api.getSubQuestions(this.surveyConceptId, this.props.stratum2, this.props.stratum3, 1)
+            .then(
+                results => {
+                    console.log(results);
+
+                    // this.props.SubQuestion.countAnalysis = results.items.filter(a => a.analysisId === 3110)[0];
+                    // console.log(results.questions.items[0].countAnalysis.results);
+                    this.setState({
+                        subQuestion: results.questions.items[0],
+                        subAnswers: results.questions.items[0].countAnalysis.results,
+                        subTitle: results.questions.items[0].conceptName
+                    });
+                    console.log(this.state.subAnswers, 'subAnswers');
+                    // console.log(this.state.subQuestion, 'Sub question');
+
+                });
+    }
+
+    render() {
+        const parcipantPercentage = this.props.countValue / this.props.partcipantCount;
+        return <React.Fragment> <div className='survey-tbl-exp-r survey-tbl-r' onClick={this.openDrawer.bind(this)}>
+            <div className='survey-tbl-d first display-body info-text survey-answer-level-1'>
+                {this.props.stratum4}
+            </div>
+            <div className='survey-tbl-r-group'>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.props.stratum3}
+                </div>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.props.countValue.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+                </div>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.props.countPercent ? this.props.countPercent.toFixed(2) : parcipantPercentage.toFixed(2)}%
+                    </div>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.hasSubQuestions === '1' ?
+                        <ClrIcon shape='caret' className='survey-row-icon' style={{ color: '#216fb4' }} dir={this.state.drawerOpen ? 'down'
+                            : 'right'} /> : <ClrIcon className='survey-row-icon' shape='bar-chart' />}
```

Comment:
Nit: put one prop per line for elements/components with more than 1-2 props:
```suggestion
                    {this.hasSubQuestions === '1' ?
                        <ClrIcon shape='caret'
                                 className='survey-row-icon'
                                 style={{ color: '#216fb4' }}
                                 dir={this.state.drawerOpen ? 'down' : 'right'} /> :
                        <ClrIcon className='survey-row-icon' shape='bar-chart' />
                    }
```